### DISCUSSION TITLE
Alert if unstable connection detected

### DISCUSSION THREAD

**Discussion by DenisTimofijuk** [michaelarnauts/home-assistant-comfoconnect#98] [Ideas]
I have been using this integration for some time now and sometimes I get strange historical charts. After investigation it seems that the connection is getting unstable and it is pulsing between 0 or actual values (see attached image for reference).
So I suggest to implement some kind of feature to track and identify if the connection is unstable and notify the user to take some action. For me it is enough to restart either the server or the device itself.
Notification would be useful as it would notify me earlier and allow me to avoid having ruined data for a long period of time.

![image](https://github.com/user-attachments/assets/934303b1-5b2a-4094-982e-3a605b7abbb9)

**Comment by TheFarmerGit** [comment]
I have had the same problem. Used the integration for about 4 weeks without any problems, but last 2 weeks I encounterd some connection issues. See attached graph of my network trafic to the Comfo connect lan c. On Thursday I removed this integration and installed the original. This is working fine since. 

Hope you can find an solution.

<img width="855" height="276" alt="image" src="https://github.com/user-attachments/assets/f481d512-ee2b-4a53-afd8-4f9e51eb6fb2" />
